app/src/main/res/layout/activity_benchmark_mode.xml

```diff
@@ -43,4 +44,30 @@
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent" />
+
+  <TextView
+      android:id="@+id/numberOfFacesView"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="180dp"
+      android:layout_marginEnd="180dp"
+      android:layout_marginBottom="18dp"
+      android:text="TextView"
```

Comment:
No need to add text here. If you want it to be displayed in preview window, change to someting more self-explanatory like a name of the field.

Reply:
+1